docs/sidebars.js

```diff
@@ -66,6 +66,10 @@ const sidebars = {
         {
             type: 'category',
             label: 'Guides',
+            link: {
```

Comment:
the sidebar is growing very large.

Reply:
This doesn't make it bigger, just makes one of the sections clickable